### ISSUE TITLE
WOQLClient.delete_graph error

### ISSUE THREAD

**Issue by mrcjhurst** [terminusdb/terminusdb-client-python#176]
**Describe the bug**
"Arguments are not sufficiently instantiated" error seen when deleting a graph.

**To Reproduce**
``` 
>>> client.create_graph('instance', 'test-one-more-graph', 'create test graph')
{'@type': 'api:CreateGraphResponse', 'api:status': 'api:success'}

>>> client.delete_graph('instance', 'test-one-more-graph', 'delete test graph')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/.../env/lib/python3.9/site-packages/terminusdb_client/woqlclient/woqlClient.py", line 722, in delete_graph
    return self.dispatch(
  File "/.../env/lib/python3.9/site-packages/terminusdb_client/woqlclient/woqlClient.py", line 1380, in dispatch
    return DispatchRequest.send_request_by_action(
  File "/.../env/lib/python3.9/site-packages/terminusdb_client/woqlclient/dispatchRequest.py", line 202, in send_request_by_action
    raise (
terminusdb_client.woqlclient.errors.APIError: {"code":500, "message":"Arguments are not sufficiently instantiated"}
```
**Expected behavior**
```
>>> client.create_graph('instance', 'test-one-more-graph', 'create test graph')
{'@type': 'api:CreateGraphResponse', 'api:status': 'api:success'}

>>> client.delete_graph('instance', 'test-one-more-graph', 'delete test graph')
{'@type': 'api:DeleteGraphResponse', 'api:status': 'api:success'}
```

**Error logs**
See above

**System information (please complete the following information):**
 
- OS: Devuan Chimera/ceres
- terminus-client-python 6.1
 - terminusdb-bootstrap Docker container

**Additional context**
a Prolog error?

**Comment by Cheukting** [comment]
It's a bug that is introduced in the rewrite (which we are doing currently for the v1.0.0 release) and it will be fixed in that release (which is SOON 😉)

**Comment by mrcjhurst** [comment]
Excellent - thank you.  Look forward to it.

**Comment by Cheukting** [comment]
@mrcjhurst the problem is resolved in v1.0.0 :-) Enjoy

**Comment by mrcjhurst** [comment]
Much better -- thank you!
